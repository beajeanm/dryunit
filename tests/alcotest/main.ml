
let () =
  Alcotest.run "Main" [
    "Secondary", [
      "test_capit", `Quick, Secondary.test_capit;
      "test_plus", `Quick, Secondary.test_plus;
      "test_capit_another4", `Quick, Secondary.test_capit_another4;
    ];
    "Primary", [
      "test_capit", `Quick, Primary.test_capit;
      "test_plus", `Quick, Primary.test_plus;
      "test_capit_another", `Quick, Primary.test_capit_another;
    ];
  ]
