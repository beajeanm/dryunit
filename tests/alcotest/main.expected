{"success":6,"failures":0}
